clients/privacy-center/app/server-utils/loadEnvironmentVariables.ts

```diff
@@ -16,7 +16,9 @@ const loadEnvironmentVariables = () => {
       "file:///app/config/config.css",
     SHOW_BRAND_LINK:
       process.env.FIDES_PRIVACY_CENTER__SHOW_BRAND_LINK === "true" || false,
-    CUSTOM_PROPERTIES: process.env.CUSTOM_PROPERTIES === "true" || true,
+    CUSTOM_PROPERTIES: process.env.CUSTOM_PROPERTIES !== "false", // default: true
```

Comment:
I'm not sure I understand the "why" behind this change, can you elaborate?

Reply:
Fixing a bug I noticed. || true will always be true even if you set it to "false". So I fix that.

It shouldn't matter too much, the default is true and I doubt anyone will set it to false. But just in case I wanted to fix it.